src/createPixelsRenderer.ts

```diff
@@ -38,6 +38,7 @@ export default function createPixelsRenderer(el: HTMLDivElement) {
       el.appendChild(maskDiv)
     }
     drawMask(w)
+    // TODO: `webkitMaskImage` is deprecated, maybe use `maskImage` instead
```

Comment:
Thanks for this comment! I agree about not changing behavior in a refactoring PR.